rfcs/0114-nixos-code-of-conduct.md

```diff
@@ -0,0 +1,112 @@
+---
+feature: NixOS Code of Conduct
+start-date: 2021-11-03
+author: @jonringer
+co-authors:
+shepherd-team: @edolstra @zimbatm @winterqt
+shepherd-leader:
+related-issues: #98 #102
+---
+
+# Summary
+[summary]: #summary
+
+The NixOS Community needs some foundation of shared values which can be used
+to determine appropriate behavior when contributing and interacting with the community.
+We currently have a [community ethos](https://nixos.org/community/index.html), but
```

Comment:
Well if singling people out for their immutable characteristics didn't work, I don't know what will…
Oh yes, it surely will help in the slightest to do it over and over again for every place and project anyone visits.
This is the opposite of cognitive behavioural therapy, which is the strongest non-psychotropic effect found in clinical psychology to date.